src/jobs/models/jobManager.js

```diff
@@ -26,6 +28,10 @@ const JOB_PLATFORM_NAME = PREDATOR_RUNNER_PREFIX + '.%s';
 module.exports.init = async () => {
     const jobPlatform = await configHandler.getConfigValue(CONFIG.JOB_PLATFORM);
     jobConnector = require(`./${jobPlatform.toLowerCase()}/jobConnector`);
+    if (jobConnector.init){
```

Comment:
implement `init` functions in all connectors and have them empty if needed

Reply:
We consulted and decided we prefer not to do it @kerenfi